apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveService.kt

```diff
@@ -91,31 +89,9 @@ class OppgaveService(
         sikreAtOppgaveIkkeErAvsluttet(hentetOppgave)
         hentetOppgave.erAttestering() && sjekkOmkanTildeleAttestantOppgave()
 
-        val eksisterendeSaksbehandler = hentetOppgave.saksbehandler?.ident
-        if (eksisterendeSaksbehandler.isNullOrEmpty() || eksisterendeSaksbehandler == Fagsaksystem.EY.navn) {
-            oppgaveDao.settNySaksbehandler(oppgaveId, saksbehandler)
-
-            // TODO: Fjerne dette. Midlertidig løsning for å støtte gammel flyt
-            if (hentetOppgave.status == Status.NY) {
-                oppgaveDao.endreStatusPaaOppgave(oppgaveId, Status.UNDER_BEHANDLING)
-            }
-        } else {
-            throw OppgaveAlleredeTildeltSaksbehandler(oppgaveId, eksisterendeSaksbehandler)
-        }
-    }
-
-    fun byttSaksbehandler(
-        oppgaveId: UUID,
-        saksbehandler: String,
-    ) {
-        val hentetOppgave =
-            oppgaveDao.hentOppgave(oppgaveId)
-                ?: throw OppgaveIkkeFunnet(oppgaveId)
-
-        sikreAtOppgaveIkkeErAvsluttet(hentetOppgave)
         oppgaveDao.settNySaksbehandler(oppgaveId, saksbehandler)
 
-        // TODO: Fjerne dette. Midlertidig løsning for å støtte gammel flyt
+        // Må ha denne for å ikke overskride attesteringstatus på oppgaver
         if (hentetOppgave.status == Status.NY) {
             oppgaveDao.endreStatusPaaOppgave(oppgaveId, Status.UNDER_BEHANDLING)
         }
```

Comment:
Det er ikke det er ikke bare IF-sjekken kommentaren refererer til, men også det at status blir satt. Grunnen til at dette ble gjort til å begynne med var fordi forrige løsning hadde det samme. Saksbehandler synes ikke det er en god løsning at tildeling setter oppgaven "under arbeid", derav TODO-en. 

Oppgaven burde få status "under arbeid" når de _faktisk_ starter arbeidet, og ikke ved tildeling .... meeeeen det får bli en oppgave til en annen gang 🤷‍♂️

Reply:
Aha, det var jeg ikke klar over. Må nesten ha den der nå for å ikke brekke attesteringsflyten for oppgaver.